src/Database/Adapter/Mongo.php

```
@@ -0,0 +1,2964 @@
+
+    /**
+     * Check if database exists
+     * Optionally check if collection exists in database
+     *
+     * @param string $database database name
+     * @param string|null $collection (optional) collection name
+     *
+     * @return bool
+     * @throws Exception
+     */
+    public function exists(string $database, ?string $collection = null): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$database'. (undefined)

(UnusedFormalParameter)